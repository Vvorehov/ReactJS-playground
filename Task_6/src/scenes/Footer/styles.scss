#footer {
  padding: 1rem 6rem;
  background-color: #434343;
}