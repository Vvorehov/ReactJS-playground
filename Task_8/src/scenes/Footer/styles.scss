#footer {
  padding: 1rem 6rem;
  background-color: #434343;
  position:fixed;
  bottom: 0;
  left: 0;
  right: 0;
}