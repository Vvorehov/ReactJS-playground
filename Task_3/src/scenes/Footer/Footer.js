import React from "react";
import { Logo } from "../Logo";
import "./styles.scss";

const Footer = () => {
  return <footer id="footer">
    <Logo siteName='netflixroulette' />
  </footer>
};

export default Footer;