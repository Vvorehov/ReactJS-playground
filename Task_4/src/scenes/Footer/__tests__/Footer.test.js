import React from "react";
import Footer from "../Footer";
import TestRenderer from 'react-test-renderer';

describe('Footer layout complete', () => {
    test('Render correct', () => {

        const component = TestRenderer.create(
            <Footer />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

