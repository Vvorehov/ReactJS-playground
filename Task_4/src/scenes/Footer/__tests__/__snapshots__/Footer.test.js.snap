// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer layout complete Render correct 1`] = `
<footer
  id="footer"
>
  <div
    className="siteLogo"
  >
    netflixroulette
  </div>
</footer>
`;
